pages/models.py

```diff
@@ -615,6 +620,29 @@ class Adopters(Page):
     parent_page_types = ['pages.HomePage']
 
 
+class Organizations(models.Model):
+    salesforce_id = models.CharField(max_length=255)
+    organization_name = models.CharField(max_length=255)
+    description = RichTextField(null=True)
+    website = models.URLField(max_length=255, null=True)
+
+    page = ParentalKey(Adopters,
```

Comment:
see how this is done with Allies on Books - you may need to subclass 'Page' to make this model. Subclassing Page does not necessarily mean it corresponds to a page, again look at allies - the way you have this, I don't think it will show up in the API